runtime/oti/j9nonbuilder.h

```diff
@@ -5918,8 +5918,9 @@ typedef struct J9JavaVM {
 	struct J9Pool *tlsPool;
 	omrthread_monitor_t tlsPoolMutex;
 	jobject vthreadGroup;
-	J9VMContinuation **globalContinuationCacheArray;
-	U_32 continuationArraySize;
+	J9VMContinuation **continuationT2Cache;
```

Comment:
`local` or `global` are better-describing terms than `T1` or `T2`, also suggesting a comment explaining the algorithms using two tiers of caches.

Reply:
T1 and T2 name change is to be consistent with xx option args and reflect the multi tier caching design.